spec/requests/api/admin/update_spec.rb

```diff
@@ -26,9 +24,7 @@
       before do
         patch "/api/v1/admin/articles/1",
         params: {
-          article: {
```

Comment:
So are we NOT wrapping the params into an `artilcle` key?

Reply:
wrapping it into object now